### PR TITLE
Adding 5s delay after WiFi firmware upload

### PR DESCRIPTION
**Pull request by SunjunKim**
# Existing behavior (2.0.0-RC3):
You know, people just press OK and don't read a message.

[IMAGE_ALT](https://youtu.be/ahnWBLOhnyw)

As seen in the video, the module stays in a *risky* state about 3-4s after pressing OK. 
Even though the message says "wait for LED to resume blinking", many will just pull out the power straight, which will brick a module.

# Improved behavior by this PR
This PR modifies web updater's  behavior to prevent possible user errors by:
1. Mark 99% at the end of an upload instead of 100%, which was giving a false cue that all the job has been done.
2. Added "Processing..." message to `completeHandler` instead of a blank, indicating a user that some work is in progress under the background.
3. Added 5 seconds delay before displaying "Update Succeeded" alert.

[IMAGE_ALT](https://youtu.be/TvWp3ntqK_Q)

As seen in the video, the RX completed rebooting with the "Update Succeeded" alert.

### CODE REVIEW

**Comment by pkendall64** [comment]
This is actually done in a slightly different way in the backpack repo
https://github.com/ExpressLRS/Backpack/pull/32/commits/339ff33244d99c1c3844da2abca2107b9cd62e75
I think it should be ported back here as well. Just waiting on approval.

The way it works in the backpack repo, is after the progress for uploading, it moves the progress-bar back to zero and counts up for 10 seconds before displaying the complete dialog.

**Comment by SunjunKim** [comment]
> The way it works in the backpack repo, is after the progress for uploading, it moves the progress-bar back to zero and counts up for 10 seconds before displaying the complete dialog.

Make sense, I'll try to backport your code here.

**Comment by SunjunKim** [comment]
Backported the ExpressLRS/Backpack@339ff33 code and it works as like this:
https://youtu.be/dFq-1xhSZa4

**Comment by CapnBry** [comment]
This looks good to me. I have no problem with getting this in for 2.0 if we want to take it even though it is a feature.

**Comment by SunjunKim** [comment]
> This looks good to me. I have no problem with getting this in for 2.0 if we want to take it even though it is a feature.

I’d like to pick this for 2.0 release. No core things changed but just to prevent user errors (which has happened a lot already)

**Review by AlessandroAU — APPROVED** [review]

**Comment by AlessandroAU** [comment]
I also agree with pick to 2.0

**Review by CapnBry — APPROVED** [review]

**Comment by CapnBry** [comment]
Tested and behaves as expected on both TX and RX side.

**Pull request merged**
